index.Rmd

```diff
@@ -89,107 +93,101 @@ Scroll down the license with enter. Accept License (i.e. type yes). Use default
+
+Let's first list the currently available environments:
 
 ```{r, eval=FALSE, engine='bash'}
-conda info --envs
+mamba info --envs
 ```
 
-Conda puts an asterisk (*) in front of the active environment. To activate the environment:
+Mamba puts an asterisk (*) in front of the active environment. To activate the environment:
 
 ```{r, eval=FALSE, engine='bash'}
-## Cross-platform (but not always working, like in our VDI, try it)
+# Cross-platform (but not always working)
 conda activate geotest
-## Linux, macOS
+
+# Linux, macOS
 source activate geotest
-## Windows
+
+# Windows
 activate geotest
 ```
 
-After this, the current environment is shown in parentheses in front of your prompt (`(geotest)$`). Note that the activated environment is only valid for the shell in which you activated it. For instance, if you close the shell window and open a new one you will have to activate it again. If you want to install a new package and are inside the conda environment, the package is installed simply with conda or pip. Generally, installing with conda is recommended. In the documentation of most Python packages the recommended way to install can be found.
+After this, the current environment is shown in parentheses in front of your prompt (`(geotest)$`). Note that the activated environment is only valid for the shell in which you activated it. For instance, if you close the shell window and open a new one you will have to activate it again.
 
-After creating a conda environment, (additional) Python packages can be installed. There are three recommended ways to install packages, which we list below. In the documentation of most Python packages the recommended way to install them can be found.
+After creating a `conda` environment, (additional) Python packages can be installed. There are three possible ways to install packages, which we list below. In the documentation of most Python packages the recommended way to install them can be found.
 
-* Using [conda](https://docs.conda.io/projects/conda/en/latest/index.html)  to install and manage packages. Conda packages are downloaded from conda channels, which are URLs to directories containing conda packages. Generally, installing with conda is the preferred method.
-* Using _pip_ to install packages and _conda_ to manage packages. _Pip_ is available for Windows, macOS and Linux. Pip can also install [binary wheels on Windows](https://www.lfd.uci.edu/~gohlke/Pythonlibs).
+* Using `mamba` to install and manage `conda` packages. This downloads `conda` packages using `conda` channels, which are URLs to directories containing the `conda` packages. **Generally, installing `conda` packages using `mamba` is the preferred method.**
+* Using `pip` to install packages and `mamba` to manage these packages. `pip` is available for Windows, macOS and Linux. `pip` can also install [binary wheels on Windows](https://www.lfd.uci.edu/~gohlke/Pythonlibs).
```

Comment:
It would be handy here to state that you should generally not install packages from `pip` in a conda environment unless it's the last resort. This is because after you use `pip` to modify an environment, you can no longer use `conda`/`mamba` to do so (trying that will break your environment, because pip does not communicate its changes to conda). Hence install packages with mamba that you can first, and only then use pip, and then never touch the environment with mamba again (delete and start fresh if you need to).